packages/public/server/src/module/Link/src/Link/Service/LinkService.php

```diff
@@ -71,6 +71,9 @@ public function associate(
 
         $this->getObjectManager()->persist($link);
 
+        $parent->getChildLinks()->add($link);
+        $child->getParentLinks()->add($link);
```

Comment:
Needed to add this for `course-page`s, because they want to create an alias using the parent directly when saving a revision. Since I want to create the entity link in the same controller handle, this needs to be injected here.